WordPress/wordcamp.org
# Blocks: Remove PanelRow wrapper from single-item controls

`PanelRow` applies flex styling, so the child elements are no longer full-width by default, the width is determined by the children elements. PanelRow is useful for aligning two controls or elements on one line, but isn't needed for single components. 

This fixes #79, #84, #86.

**To test**

- Add any of the blocks (this touches Organizers, Sessions, and Speakers)
- Check that the controls in the sidebar are displayed correctly, don't overlap or have extra space.
